src/general/block-endpoint-edit-role/src.rego

```diff
@@ -0,0 +1,32 @@
+package blockendpointeditrole
+
+violation[{"msg": msg}] {
+    input.review.object.metadata.name == "system:aggregate-to-edit"
+    endpointRule(input.review.object.rules[_])
+    msg := "ClusterRole system:aggregate-to-edit should not allowed endpoint permissions"
```

Comment:
nit: "endpoint permissions" -> "endpoint edit permissions"

Reply:
nit: s/should not allowed/should not allow

Reply:
thx and fixed